server/src/main/java/org/elasticsearch/common/settings/Settings.java

```diff
@@ -529,7 +529,7 @@ public static Settings readSettingsFromStream(StreamInput in) throws IOException
         return builder.build();
     }
 
-    public static void writeSettingsToStream(Settings settings, StreamOutput out) throws IOException {
+    public static void writeSettingsToStream(StreamOutput out, Settings settings) throws IOException {
```

Comment:
Could you extract this change into a dedicated commit - or better PR - with an explanation why this order is better?

Reply:
Extracted to https://github.com/crate/crate/pull/15608, Thank you